cases/pysb/0014/_observables.tsv

```diff
@@ -0,0 +1,2 @@
+observableId	observableFormula	noiseFormula
+obs_a	A	noiseParameter1_obs_a + noiseParameter2_obs_a
```

Comment:
Noise formula without states could in principle be added to the pysb model during import. How do we want to handle that? All need to be defined in the pysb model beforehand? Maybe more consistent... How do we do the mapping of observable and noise Expression in the pysb model?

Reply:
I wouldn't expect anyone to define noise formulas in pysb. I still don't think it is sane to specify state dependent sigmas, observable dependent should be sufficient. What are the current restritions for sbml?

For petab specified observables, it should be doable to import the model and then add the expression via something like
```
Expression(
    'observableId', 
    sp.sympify(
        'observableFormula', 
        locals=[comp for comp in pysb.components if isinstance(comp, sp.Symbol)]
    )
)
```
And similar operations should be applicable for noise Formula etc, just have to be careful about order in which they are added to the model.

Reply:
> observable dependent should be sufficient

Works for me. Not a serious restriction...

> What are the current restritions for sbml?

There are pretty much none

> import the model and then add the expression via something like

Let's try. Thanks for the suggestion.